src/NServiceBus.AcceptanceTests/Core/Encryption/When_using_encryption_with_custom_service.cs

```diff
@@ -9,6 +9,8 @@
     using NServiceBus.Pipeline;
     using NUnit.Framework;
 
+    // disable obsolete warnings. Tests will be removed in next major version
+    #pragma warning disable CS0618
     public class When_using_encryption_with_custom_service : NServiceBusAcceptanceTest
```

Comment:
I think you need to close the scope of the pragma so that it doesn't apply to next files if I recall correctly